buildpack.md

```diff
@@ -204,26 +234,24 @@ Before the next re-build:
 
 A buildpack MAY specify that a `<layers>/<layer>/` directory is a cached layer by placing `cache = true` in `<layers>/<layer>.toml`.
 
-If a cache is provided the lifecycle:
+If a cache is provided, the lifecycle:
 - SHOULD store all cached layers after a successful build.
 - SHOULD store the Layer Content Metadata associated with each layer so that it can be recovered using the layer Diff ID
 
 Before the next re-build:
-- The lifecycle:
-  - MUST either restore the entire `<layers>/<layer>.toml` file and corresponding `<layers>/<layer>/` directory from any previous build to the same paths or
-  - MUST restore neither the `<layers>/<layer>.toml` file nor corresponding `<layers>/<layer>/` directory.
+- The lifecycle MUST do both or neither of the following:
+  - Restore Layer Content Metadata to `<layers>/<layer>.toml`, excluding the `launch`, `build`, and `cache` booleans.
+  - Restore layer contents to the `<layers>/<layer>/` directory.
+
+#### Ignored Layers
 
-#### Other Layers
+Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, existing for the duration of a single build.
 
-The lifecycle:
-- If `build = false`
-  - MUST NOT make the layer available to subesequent buildpacks.
-- If `launch = false`
-  - MUST NOT include the layer or the Layer Content Metadata in the OCI image.
-- If `cache = false`
-  - MUST NOT store the layer or the Layer Content Metadata in the cache.
+At the end of each individual buildpack's build phase:
+- The lifecycle:
+  - MUST rename `<layers>/<layer>/` to `<layers>/<layer>.ignore/` for all layers where `launch = false`, `build = false`, and `cache = false`, in order to prevent subsequent buildpacks from accidentally depending on an ignored layer.
 
-Layers marked `launch = false`, `build = false`, and `cache = false` behave like temporary directories, available only to the authoring buildpack, that exist for the duration of a single build.
+The lifecycle restores all layers from previous builds as ignored layers. Buildpacks MUST set `launch`, `build`, and/or `cache` in `<layers>/<layer>.toml` to use a restored layer.
```

Comment:
Is an "ignored" layer one that has `build`, `launch`, and `cache` set to false?

Reply:
Yes. The first line in the "gnored layers" sections attempts to communicate that.

Reply:
@hone, I agree that this sentence might be a bit confusing.
@ekcasey, the lifecycle restores all layers from previous builds and sets the 3 flags to false, but it doesn't rename `<layers>/<layer>/` to `<layers>/<layer>.ignore/`. So maybe it's not very accurate to write that the layers are restored as `ignored layers`.
What do you think?

Reply:
This paragraph seems pretty well covered in the `Reusing Layers`. I feel like we could just remove it.

Reply:
After talking with @ekcasey and @natalieparellano, we decided to delete this line.
Please see https://github.com/buildpacks/spec/pull/209/commits/3cd0f8be71cf90f563cbaa788ad1758f7d00f3ad.